optax/_src/alias.py

```diff
@@ -431,6 +431,29 @@ def noisy_sgd(
   )
 
 
+def online_newton_step(learning_rate: ScalarOrSchedule,
+                       eps: float) -> base.GradientTransformation:
+  # pylint: disable=line-too-long
+  """Online Newton Step optimizer.
+  (see the description of the ONS in Fig. 2 p. 176 of the reference below).
+
+  References:
+    Hazan, E., Agarwal, A. and Kale, S., 2007. Logarithmic regret algorithms for online convex optimization. Machine Learning, 69(2-3), pp.169-192 : https://link.springer.com/content/pdf/10.1007/s10994-007-5016-8.pdf
+
+  Args:
+    learning_rate: this is a fixed global scaling factor.
+    initial_accumulator_value: initialisation for the accumulator.
+    eps: A floating point value to avoid zero denominator.
+
+  Returns:
+    the corresponding `GradientTransformation`.
+  """
```

Comment:
(If we do end up keeping the long line we should enable the warning again immediately after the docstring.)